### PR TITLE
Changed fabric.mod.json

### PR DESCRIPTION
**Pull request by DragonEggBedrockBreaking**
Changed fabric.mod.json to make it so that carpet extra displays as a sub-part of carpet in modmenu, as Carpet TIS Additions has already done.

### CODE REVIEW

**Comment by altrisi** [comment]
I think it would be better to be the other way around (TIS being outside), since it's technically a different mod (Ik, Fallen wants TIS addition to be parented), and it's more visible this way.

**Comment by DragonEggBedrockBreaking** [comment]
"since it's technically a different mod"
Yet it is still an extension and adds features on to carpet.

**Comment by DragonEggBedrockBreaking** [comment]
Also, other (extensions) are adding this, so clearly other people agree: Lunnar Carpet Addons and Carpet TIS Addition already have it, and my PRs on scarpet additions and carpet addons got merged.

**Review by altrisi — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,352 +1,6 @@
...
-* Type: `boolean`  
-* Default value: `false`  
-* Required options: `true`, `false`  
-* Categories: `EXPERIMENTAL`, `EXTRAS`  
-  
+Carpet Extra - Unofficial Fork
+============
 
+### IF YOU WANT CARPET EXTRA, GO HERE: https://github.com/gnembon/carpet-extra
+THIS IS A FORK OF CARPET EXTRA FOR MY TESTING PURPOSES. DO NOT USE THIS FORK. I HAVE ABSOLUTELY NO IDEA WHAT I AM DOING SO DON'T LOOK TO ME.
```
Comment (altrisi) [inline README.md:5]: Hm... No?

**Review by altrisi — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (DragonEggBedrockBreaking) [reply README.md:5]: yeah, sorry, that's my fork
imma undo that and put it into a branch (of my fork) unrelated to this pr
i fucked up, sorry

Comment (DragonEggBedrockBreaking) [reply README.md]: ok, should be fixed now
sorry abt the mistake

Comment (altrisi) [reply README.md]: As a recommendation for the future, it's always better to make PRs from new branches, so they don't mess up and are easier to update with upstream.

**Comment by DragonEggBedrockBreaking** [comment]
:-)

**Pull request merged**